index.html

```diff
@@ -29,352 +32,374 @@
-          website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don’t hestiate to contact
-          me.
-        </p>
-      </div>
-      <div class="base-text">LET'S CONNECT</div>
-      <div class="media-icons">
-        <ul>
-          <li>
-            <a href=""><img src="images/vector.svg" alt="image" /></a>
-          </li>
-          <li>
-            <a href=""><img src="images/in.svg" alt="image" /></a>
-          </li>
-          <li>
-            <a href=""><img src="images/vectorm.png" alt="image" /></a>
-          </li>
-          <li>
-            <a href=""><img src="images/vectorhead.png" alt="image" /></a>
-          </li>
-          <li>
-            <a href=""><img src="images/vectorhand.png" alt="image" /></a>
-          </li>
-        </ul>
-      </div>
-    </div>
-      <section id = "works">
-       
-      <div id="section-1">
-         <!----Markup for desktop card-1-->
-         <div class = "s1-container-dt">
-          <div class = "img-container">
-   <img src = "images/desktop-section-1.png" alt = "The book"/>  
-              </div>
-              <div class = "card-text">
-          <h1>Tonic</h1>
-          <div class = "list-container">
-            <ul>
-              <li><a href="" class="text">CANOPY</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Back End Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <div class = "base-links">
-            <ul>
-              <li><a href="" class="base-link-text1">html</a></li>
-              <li><a href="" class="base-link-text2">css</a></li>
-              <li><a href="" class="base-link-text3">javascript</a></li>
-            </ul>
-          </div>
-          <div class = "base-button">
-          <a href = "#">See Project</a>
-          </div>
-          </div>
-  
-  </div>
-        <!---End of desktop card 1 Mark up-->
-        <div class="s1-container">
-          <div class="image-container"></div>
-          <div class = "content">
-          <h1>Tonic</h1>
-          <div class="list-container">
-            <ul>
-              <li><a href="" class="text">CANOPY</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Back End Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div class = "para">
+        <div class="middle-text">
+          <h1>I'm Dickson Glad to see you.</h1>
```

Comment:
- [ ] I think the text heading here is not placed right because you have `h2` tag before `h1` and I think you should follow the headings for accessibility Having a unique H1 tag would be helpful in regards to Google recognizing unique relevance between different pages. The H1 tags are one of the most significant SEO elements on the page besides the title tag.

Hint: you can use one `h1` but then use `h2` or `h3` to follow the headings and in order, if you want to be darker just use font-weight for that 😉